webcompat/form.py

```diff
@@ -100,6 +114,15 @@ def get_problem(category):
     return u'Unknown'
 
 
+def get_browser_test(category):
```

Comment:
@karlcow not clear about this function, form_object.get('problem_category') from 
https://github.com/webcompat/webcompat.com/blob/master/webcompat/form.py#L192

is it getting the current value selected on the webpage form?